lib/user.rb

```diff
@@ -0,0 +1,22 @@
+class User
+  attr_reader :name, :jokes
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << (joke)
+  end
+
+  def tell(friend, joke)
+    friend.learn(joke)
```

Comment:
NICE! Great job seeing that a friend was an instance of the user class, so you were able to use the `learn` method you'd already written to do the work.  Good utilization of object interaction.